bootstrap/js/contact.js

```diff
@@ -0,0 +1,75 @@
+//Add an event listener for the form submission to create a new user
+
+const form = document.querySelector("form");
+
+
+const formEvent = form.addEventListener('submit', event => {
+   event.preventDefault(); 
+   const username = document.querySelector("#username").value;
+   const email = document.querySelector("#email").value;
+   const message = document.querySelector("#message").value;
+
+   const user = { username, email, message };
+
+
+   createUser(user);
+});
+
+
+
+const createUser = (user) => {
+    let postObject = {...user}
+axios({
+    method: 'post',
+    url: ' https://gg-web-api.herokuapp.com/users/',
+    data: {
+      "username": `${postObject.username}`,
+      "email": `${postObject.email}`,
+      "message": `${postObject.message}`
+    }
+  });
+
+}
+
+
+//flash message when a user post data to the endpoint 
+(function($) {
```

Comment:
@Ugerpaul17 , I've run the website and submitted a user but do not see any message popup from this. Please confirm that this is working on your end. You can attach a gif on this PR in the comment section using this => https://www.screentogif.com/

Reply:
it is working on my end, I was unable to install screentogif because my operating system does not meet the requirement for .NET 
![Screenshot (25)](https://user-images.githubusercontent.com/25029435/77909054-4ade5600-7295-11ea-9049-11267d70d6fc.png)